sdk/python/kfp/_client.py

```diff
@@ -125,7 +126,9 @@ def _load_config(self, host, client_id, namespace, other_client_id, other_client
 
     # Obtain the tokens if it is IAP or inverse proxy.
     # client_id is only used for IAP, so when the value is provided, we assume it's IAP.
-    if client_id:
+    if other_token:
```

Comment:
can u add some comments on what's the use case for other_token?

Reply:
good point, I planned to put it to doc but keep quick sample here is better.

(don't know why PR routed to your queue, hope no interruption)